x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/editor_frame.scss

```diff
@@ -0,0 +1,74 @@
+.lnsPage {
+  padding: 0;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  overflow: hidden;
+}
+
+.lnsSidebar {
+  margin: 0;
+  flex: 1 0 18%;
+  min-width: ($euiSize * 16);
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+.lnsSidebar--right {
+  @include euiScrollBar;
+  overflow: hidden auto;
+  padding: $euiSize;
+  min-width: ($euiSize * 18);
+}
+
+.lnsPageBody {
+  @include euiScrollBar;
+  padding-top: $euiSize;
+  padding-bottom: $euiSize;
+  overflow: hidden auto;
+
+  &:first-child {
+    padding-left: $euiSize;
+  }
+
+  .lnsPageContent {
+    padding: 0 !important;
```

Comment:
Is it worth using these components if they require this level of override?

Reply:
I think so - the important statements were not necessary and it adds other things (e.g. box shadow)